GAN/VanillaGAN-Torch/README.md

```diff
@@ -0,0 +1,134 @@
+# how to run the code
+- on colab notebook :
+```python
+!git clone link-to-repo
+%run main.py --k_steps 5 --epoch 50 #customize according to what you want!
+```
+- on  a local machine :
+```bash
+git clone link-to-repo
+```
+```python
+python3 main.py --k_steps 5 --epoch 50 # customize as you want!
+```
```

Comment:
format this part according to resnet README

Reply:
what should i exactly do here?

Reply:
https://github.com/pclubiitk/model-zoo/blob/master/image_classification/ResNet-PyTorch/README.md  similar to this

Reply:
https://github.com/pclubiitk/model-zoo/blob/master/image_classification/ResNet-PyTorch/README.md (similar to this)